src/main/java/com/epam/esm/util/PropertyReader.java

```diff
@@ -0,0 +1,44 @@
+package com.epam.esm.util;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyReader {
+    private static final Properties properties = new Properties();
+    private static PropertyReader instance = null;
+
+    public static Properties getProperties() {
+        if (instance == null) {
+            loadProperties();
+            instance = new PropertyReader();
+        }
+        return properties;
+    }
+    /**
+     * Reading properties from application.properties
+     */
+    public static void loadProperties() {
+        String propertiesFileName = "/Users/andrew_wannasesh/folders/" +
```

Comment:
All project filepaths should rely on classpath. Usage of system paths will make it impossible to run the application on other computers.